hedera-node/hedera-evm/build.gradle.kts

```diff
@@ -15,12 +15,10 @@
  */
 
 plugins {
-    id("com.hedera.hashgraph.conventions")
-    id("com.hedera.hashgraph.evm-maven-publish")
+    id("com.hedera.gradle.evm")
```

Comment:
I'm not complaining about renaming this module, I think it's a good idea, but where'd it come from?  Is this part of the move to make it easier to get the EVM module over to the mirror node?  Or just consistency?  Or something else?

Reply:
This is for consistency and backward compatibility. EVM is already published now with a different `group` and we want to keep that. (See other discussions with Nathan on this PR.)